src/terminal.go

```diff
@@ -1758,6 +1759,46 @@ func (t *Terminal) executeCommand(template string, forcePlus bool, background bo
 	cleanTemporaryFiles()
 }
 
+func (t *Terminal) executeChangeQuery(template string) {
+	valid, list := t.buildPlusList(template, false)
+	if !valid {
+		return
+	}
+	command := t.replacePlaceholder(template, false, string(t.input), list)
+	cmd := util.ExecCommand(command, false)
+	newQuery := []rune{}
+	t.executing.Set(true)
+	cmd.Stdin = tui.TtyIn()
```

Comment:
I put this line in because it's in the `executeCommand` func above. I'm not exactly sure what it does but am guessing that it provides user input to interactive commands being executed. I haven't yet tried out an interactive command in the new `change-query(...)` action, though.